tests/test_validator_integration.py

```
@@ -0,0 +1,975 @@
+class TestConfigConsistency:
+    """All config files in configs/ should have the same top-level sections
+    and the same set of training fields."""
+
+    CONFIG_FILES = [
+        "config.yaml",
+        "config_span.yaml",
+        "config_token.yaml",
+        "config_decoder.yaml",
+        "config_biencoder.yaml",
+        "config_relex.yaml",
+    ]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)